build/synapse/Dockerfile

```diff
@@ -9,10 +9,8 @@ RUN \
 
 RUN /synapse-venv/bin/pip install psycopg2 coincurve pycryptodome
 
-# Temporary patch until raiden-network/raiden#4634 is resolved
-RUN sed -i -E 's/^MAX_DISPLAYNAME_LEN = 100$/MAX_DISPLAYNAME_LEN = 200/' /synapse-venv/lib/python3.7/site-packages/synapse/handlers/profile.py
```

Comment:
This is in the recent synapse version?

Reply:
Yes, that was left over from when we were using `1.3.x`. The change to 200 a char limit has been in since `1.4.0`